data.sql

```diff
@@ -98,4 +98,44 @@ SET
     owner_id = 'Dean Winchester'
 WHERE
     name = 'Angemon'
-    OR name = 'Boarmon';
\ No newline at end of file
+    OR name = 'Boarmon';
+
+INSERT INTO
+    vets (name, age, date_of_graduation)
+VALUES
+    ('William Tatcher', 45, 'Apr 23, 2000'),
+    ('Maisy Smith', 26, 'Jan 17,2019'),
+    ('Stephanie Mendez', 64, 'May 4,1981'),
+    ('Jack Harkness', 38, 'Jun 8,2008');
```

Comment:
- [ ] Kindly use numbers for the date as it is used so that the requirement would be followed, same thing for others.